modules/RockRuby.cmake

```diff
@@ -21,6 +21,10 @@
 # endif()
 #
 # 
+# If installed to a target directory set the following to true to
+# make it project specific
+# set(RUBY_USE_PROJECT_INSTALL_PREFIX TRUE)
```

Comment:
Ok, I reecked and it might indeed be the wrong fix - so for vizkit the current situation is the following: the extension is required as
    require 'vizkit/vizkittypelib'

So I guess that should be fixed to "require 'vizkittypelib' then" and same for TypelibQtAdapter